common/src/main/java/org/corfudb/common/metrics/micrometer/registries/LoggingMeterRegistryWithHistogramSupport.java

```diff
@@ -65,22 +65,22 @@ public LoggingMeterRegistryWithHistogramSupport(LoggingRegistryConfig config,
     }
 
     Stream<String> writeCounter(Meter.Id id, double count) {
-        if (Double.isFinite(count)) {
+        if (Double.isFinite(count) && count != 0) {
             return Stream.of(influxLineProtocol(id, "counter", Stream.of(new Field("value", count))));
         }
         return Stream.empty();
     }
 
     Stream<String> writeGauge(Meter.Id id, Double value) {
-        if (Double.isFinite(value)) {
+        if (Double.isFinite(value) && value != 0) {
```

Comment:
Guage can go up and down right, so is this ok for printing only non-zero value?

Reply:
I agree, unlike every other meter (timer, distribution), gauges report current values and not rates. 0 is a valid current value to be reported.